back/spec/acceptance/projects_spec.rb

```diff
@@ -987,7 +986,6 @@
                   0,
                   0,
                   0,
-                  ideation_response.budget,
```

Comment:
I would think that it could still be meaningful to know the budget in an ideation phase? But fine for me if we decided to drop it of course.

Reply:
Yeah maybe you're right there. Will add it back in.